lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2460,6 +2460,159 @@ definitions:
+        example: "ms480"
+      a2_threshold_rsrp:
+        type: integer
+        format: int32
+        minimum: 0
+        maximum: 97
+        example: 5
+      lte_a2_threshold_rsrq:
+        type: integer
+        format: int32
+        example: 6
+      lte_a2_threshold_rsrp_irat_volte:
+        type: integer
+        format: int32
+        example: 7
+      lte_a2_threshold_rsrq_irat_volte:
+        type: integer
+        format: int32
+        example: 8
+      a3_offset:
+        type: integer
+        format: int32
+        minimum: -30
+        maximum: 30
+        example: 10
+      a3_offset_anr:
+        type: integer
+        format: int32
+        example: 10
+      a4_threshold_rsrp:
+        type: integer
+        format: int32
+        example: 50
+      lte_intra_a5_threshold_1_rsrp:
+        type: integer
+        format: int32
+        minimum: 0
+        maximum: 97
+        example: 50
+      lte_intra_a5_threshold_2_rsrp:
+        type: integer
+        format: int32
+        minimum: 0
+        maximum: 97
+        example: 51
+      lte_inter_anr_a5_threshold_1_rsrp:
+        type: integer
+        format: int32
+        minimum: 0
+        maximum: 97
+        example: 55
+      lte_inter_anr_a5_threshold_2_rsrp:
+        type: integer
+        format: int32
+        minimum: 0
+        maximum: 97
+        example: 50
+      b2_threshold1_rsrp:
+        type: integer
+        format: uint32
+        minimum: 0
+        maximum: 97
+        example: 8
+      b2_threshold2_rsrp:
+        type: integer
+        format: int32
+        minimum: -5
+        maximum: 91
+        example: 8
+      b2_geran_irat_threshold:
+        type: integer
+        format: uint32
+        minimum: 0
+        maximum: 63
+        example: 8
+      qrxlevmin_sib1:
+        type: integer
+        format: int32
+        minimum: -70
+        maximum: -22
+        example: 8
+        default: -23
+      qrxlevminoffset:
+        type: integer
+        format: uint32
+        minimum: 1
+        maximum: 8
+        example: 8
+        default: 6
+      s_intrasearch:
+        type: integer
+        format: uint32
+        minimum: 0
+        maximum: 31
+        example: 8
+      s_nonintrasearch:
+        type: integer
+        format: uint32
+        minimum: 0
+        maximum: 31
+        example: 8
+      qrxlevmin_sib3:
+        type: integer
+        format: int32
+        minimum: -70
+        maximum: -22
+        example: 8
+        default: -23
+      reselection_priority:
+        type: integer
+        format: uint32
+        minimum: 1
+        maximum: 65535
+        example: 8
+      threshservinglow:
+        type: integer
+        format: uint32
+        example: 8
+      ciphering_algorithm:
+        type: string
+        example: "128-EEA1,128-EEA2"
+        default: "128-EEA1,128-EEA2"
+      integrity_algorithm:
+        type: string
+        example: "128-EIA1,128-EIA2"
+        default: "128-EIA1,128-EIA2"
```

Comment:
I think setting default values on this level probably doesn't work (see this [issue](https://github.com/go-swagger/go-swagger/issues/1393)). Instead you could set them one level higher, as described in the issue.

If it works you will see (in the swagger GUI) a line "Default value : ..."  next to the respective value  of the swagger endpoint.

Reply:
![image](https://user-images.githubusercontent.com/87301024/154631231-a7e1c4c6-283e-43a6-9e49-44c3a7feb16a.png)
The default configuration parameter has been canceled. I don't think it is necessary to configure the default value in this place

Reply:
Alright!